packages/local_auth/local_auth/README.md

```diff
@@ -252,6 +252,37 @@ types (such as face scanning) and you want to support SDKs lower than Q,
 _do not_ call `getAvailableBiometrics`. Simply call `authenticate` with `biometricOnly: true`.
 This will return an error if there was no hardware available.
 
+#### Android theme
+
+On Android, ensure that the `LaunchTheme` of your app has a parent style with the `Theme.AppCompat.Light` theme; otherwise the app may crash. To do that go to `android > app > src > main > res > values > styles.xml` and look for the style with name `LaunchTheme` (Notice that `LaunchTheme` must be referenced in the `AndroidManifest.xml` file). Then, change the parent style with the new value and you should have set up something like this:
```

Comment:
Why is `.Light` the only option here? Why not just `Theme.AppCompat`?

Reply:
I made some research and test all variants of the `Theme.AppCompat`, and these were the results:

Using `Theme.AppCompat` alone results in a dialog with weird (no contrast) theme:
<img width="283" alt="android-6-appcompat" src="https://user-images.githubusercontent.com/64153497/216766140-934f7eb0-c16c-48ba-a344-5e2781925a02.png">

Therefore, only two options left from the documentation:
- **Light**
- **Dark**

However, since API 29 there is an extra option: **DayNight**, which enables light/dark modes switching when the user changes the theme from the device's settings.

To check the later, I set up the `Theme.AppCompat.DayNight` theme and these are the results:
-  Choosing light mode from device's settings
<img width="314" alt="android-11-appcompat-daynight-light" src="https://user-images.githubusercontent.com/64153497/216766861-9b9e54d3-e28b-4cce-a306-ef06f14eff5a.png">

- Chossing dark mode from device's settings
<img width="315" alt="android-11-appcompat-daynight-dark" src="https://user-images.githubusercontent.com/64153497/216766869-6a8d180d-60e1-43e5-9b7a-e0bd0b739e25.png">

The good thing is that even the `Theme.AppCompat.DayNight` theme has only sense from API 29, it maps the available mode of lower API versions. The following is an example of using `DayNight` on API 23 (Android 6):
<img width="282" alt="android-6-appcompat-daynight" src="https://user-images.githubusercontent.com/64153497/216766779-7e37946a-edd3-4fbc-b6e1-895333e714ec.png">

As the options of `AppCompat` are related to a usecase, I updated the doc by suggesting to apply one of the AppCompat options. And I gave the `DayNight` example:

You need to update the `LaunchTheme` parent style with a valid `Theme.AppCompat`
theme to be compatible with **Android version 8 and below**, otherwise the app
crashes for those versions. For example, use `Theme.AppCompat.DayNight` to
enable light/dark modes for the biometric dialog.